hyperspy/io_plugins/bcf.py

```diff
@@ -1029,7 +1104,7 @@ def bcf_imagery(obj_bcf, instrument=None):
                      # possibilities to get such parameter from bruker
                      # or some SEM's'
                      'beam_energy': obj_bcf.header.sem.hv,
-                     'tilt_stage': obj_bcf.header.stage.tilt_angle,
+                     'tilt_stage': obj_bcf.header.stage.tilt_stage,
                      'stage_x': obj_bcf.header.stage.x,
                      'stage_y': obj_bcf.header.stage.y
```

Comment:
I could simply not pass the None, I don't see the benefits getting one more abstraction layer. It is enought bcf is like onion... Actually I find it quite hard to follow the use of 'DictionaryTreeBrowser' in other format implementation. I understand the point I will do something about it, but I am not sure mapping is ok.